doc/source/rllib/package_ref/index.rst

```diff
@@ -0,0 +1,19 @@
+.. include:: ../we_are_hiring.rst
+
+.. _rllib-reference-docs:
+
+RLlib API Reference
+===================
+
+.. tip:: We'd love to hear your feedback on using RLlib - `ask your questions or give feedback at <https://discuss.ray.io>`_!
+
+This section contains an overview of RLlib's package- and API reference.
+If you think there is anything missing, please open an issue on `Github`_.
+
+.. _`GitHub`: https://github.com/ray-project/ray/issues
+
+.. toctree::
+   :maxdepth: 2
+
+   env.rst
+   ../../rllib-package-ref.rst  .. TODO: Eventually, move all reference pages into this dir here (doc/source/rllib/package_ref/)
```

Comment:
i don't think the bottom rendered properly

Reply:
Yeah, I'll check ...

Reply:
done